scipy/signal/signaltools.py

```diff
@@ -2505,7 +2505,7 @@ def vectorstrength(events, period):
     return strength, phase
 
 
-def detrend(data, axis=-1, type='linear', bp=0):
+def detrend(data, axis=-1, type='linear', bp=0, overwrite_data=True):
```

Comment:
you cannot change the current behavior, so default should be False

Reply:
Got mixed up when I changed it from "copy" to "overwrite_data." It shouldn't have changed! You're right.